src/views/PrivacyView.vue

```diff
@@ -78,39 +71,22 @@ const navigateToHome = () => {
 
     <div class="col-12">
       <Card>
-        <template #title> Hosting </template>
+        <template #title> {{ $t('privacy.hosting.title') }}</template>
         <template #content>
           <p class="p-card-content-header">
-            Unser Hoster erhebt in sog. Logfiles folgende Daten, die Ihr Browser übermittelt:
+            {{ $t('privacy.hosting.explanation.1') }}
           </p>
           <p>
-            Ihre IP-Adresse, Ihr Browsertyp und Ihre Browserversion, Ihr Betriebssystem, die
-            Internetseite, von der Sie auf unsere Website gelangt sind, die Unterwebseiten, welche
-            über ein zugreifendes System auf unserer Internetseite angesteuert werden, das Datum und
-            die Uhrzeit eines Zugriffs auf unsere Website und Ihr Internet-Service-Provider.
+            {{ $t('privacy.hosting.explanation.2') }}
           </p>
           <p>
-            Wir benötigen die vorgenannten Daten, um die Inhalte unserer Internetseite korrekt
-            auszuliefern und die dauerhafte Funktionsfähigkeit unserer informationstechnologischen
-            Systeme und der Technik unserer Internetseite zu gewährleisten. Die gespeicherten Daten
-            werden nur zu statistischen Zwecken ausgewertet. Rückschlüsse auf Ihre Person werden aus
-            den Daten nicht gezogen, insbesondere werden die Server-Logfiles getrennt von Ihren
-            sonstigen personenbezogenen Daten gespeichert. Personenbezogene Nutzerprofile können
-            nicht gebildet werden.
+            {{ $t('privacy.hosting.explanation.3') }}
           </p>
           <p>
-            Die Verarbeitung der Daten erfolgt auf Grundlage von Art. 6 I lit. b) DSGVO. Nach dieser
-            Vorschrift ist die Verarbeitung personenbezogener Daten zur Erfüllung eines Vertrags,
-            dessen Vertragspartei Sie sind, oder zur Durchführung vorvertraglicher Maßnahmen
-            erforderlich. Mit der Nutzung unserer Seite nehmen Sie unser Angebot auf Abschluss eines
-            entsprechenden (unentgeltlichen) Nutzungsvertrages stillschweigend an. Im Übrigen ist
-            die Verarbeitung auch über Art. 6 I lit. f) DSGVO gerechtfertigt, da wir ein
-            berechtigtes Interesse an der Verarbeitung haben und Ihre Interessen, Grundrechte und
-            Grundfreiheiten nicht überwiegen. Das Interesse besteht darin, unser Webangebot
-            ordnungsgemäß auszuliefern und die Funktionsfähigkeit unserer Systeme zu schützen.
+            {{ $t('privacy.hosting.explanation.4') }}
           </p>
           <p class="p-card-content-header">
-            Wir setzen für die Zurverfügungstellung unserer Website folgenden Hoster ein:
+            {{ $t('privacy.hosting.explanation.5') }}
           </p>
           <p>
             Hochschulrechenzentrum HTW Berlin<br />
```

Comment:
It’s not necessary to translate the text here, but it would be better to manage and maintain all text consistently in the JSON file for easier updates and uniformity.